packages/visual-dev/src/components/Alerts/Alerts.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import styled from 'styled-components'
+
+import * as Styles from './Styles'
+
+interface AlertProps {
+	type: "critical" | "warning" | "success" | "info"
+	title?: string
+	children: React.ReactNode
+}
+
+const AlertDiv = styled.div<Pick<AlertProps, 'type'>>`
+	${Styles.base}
+	${props => Styles[props.type]}
+`
+
+const SVG = {
```

Comment:
This might be good motivation to update the `Icon` component to include more icons, then we can avoid hardcoded svg code in each component that uses different icons. Not blocking now but definitely should be cleaned up at a later stage. Maybe the `Icon` component could be your next component to work on @kutaycinar ?